docker-entrypoint.sh

```diff
@@ -15,18 +15,29 @@ env | grep -iv PASS | sort
 
 if [ "$1" = 'dev' ]; then
     echo "[Run] developing site"
+
     gulp_run
-    cd /app/site && exec bundle exec jekyll serve --host=0.0.0.0
+
+    cd /app/site 
+    bundle 
+
+    exec jekyll serve --host=0.0.0.0
 fi
 
 if [ "$1" = 'releasetarball' ]; then
     echo "[Run] building production tarball"
+
     gulp_run
-    cd /app/site && bundle exec jekyll build
-    cd /app/site/_site && exec tar czf /build/${TARBALL_NAME}-${TARBALL_GIT_TAG}.tar.gz .
+
+    cd /app/site
+    bundle
+    jekyll build
+
```

Comment:
same thing here, `bundle jekyll build` should be one line (and the exec should stay in there, it's an argument to `bundle` not a shell `exec`)